gradoop-temporal/src/main/java/org/gradoop/temporal/model/impl/operators/aggregation/functions/MaxEdgeTime.java

```diff
@@ -28,11 +28,21 @@
   /**
    * Creates an instance of the {@link MaxEdgeTime} aggregate function.
    *
-   * @param aggregatePropertyKey The aggregate property key.
    * @param dimension            The time dimension to consider.
    * @param field                The field of the time-interval to consider.
    */
-  public MaxEdgeTime(String aggregatePropertyKey, TimeDimension dimension, TimeDimension.Field field) {
-    super(aggregatePropertyKey, dimension, field);
+  public MaxEdgeTime(TimeDimension dimension, TimeDimension.Field field) {
+    super(dimension, field);
+  }
+
+  /**
+   * Creates an instance of the {@link MaxEdgeTime} aggregate function.
+   *
+   * @param dimension            The time dimension to consider.
+   * @param field                The field of the time-interval to consider.
+   * @param aggregatePropertyKey The aggregate property key.
```

Comment:
I know we used this param description, but maybe make it a bit more meaningful, like "The key of the property where the aggregated result is saved." os something like that

Reply:
After merging, we have to check all our other repos that use the snapshot of gradoop to change the API call there, e.g. graph explorer